packages/apps/plugins/plugin-outliner/src/components/Outliner/Outliner.stories.tsx

```diff
@@ -25,7 +25,7 @@ const Story = ({
   const [root] = useState<TreeItemType>(
     create<TreeItemType>({
       id: 'root',
-      text: create(TextV0Type, { content: '' }),
+      content: '',
```

Comment:
do we need to explicitly set `''`?

Reply:
Yes, because it's a required field, but my expectation is that we'll be able to define defaults for fields eventually to remove this tedium.